server/leader.go

```diff
@@ -298,7 +298,17 @@ func (s *Server) watchLeader(leader *pdpb.Member) {
 
 	ctx, cancel := context.WithCancel(s.serverLoopCtx)
 	defer cancel()
-
+	err := s.reloadConfigFromKV()
+	if err != nil {
+		log.Error("reload config failed:", err)
+		return
+	}
+	err = s.cluster.regionSyncer.statSyncerWithLeader(leader.GetClientUrls()[0])
```

Comment:
Should not the first address. Maybe part of addrs can be accessed and the valid address is not in the first place.